frontend/src/pages/AboutUs/OurTeamTab.tsx

```diff
@@ -315,7 +317,7 @@ function OurTeamTab() {
           <Grid item>
             <Grid container className={styles.GridSubRow}>
               {GOOGLE_FELLOWS.map((fellow) => {
-                return (
+                return fellow.link == null ? (
```

Comment:
Maybe change to `fellow.link === null` ?  @kkatzen ?

Reply:
I can change it

Reply:
That actually doesnt compile